designer/src/components/condition.tsx

```diff
@@ -623,7 +623,9 @@ const BooleanConditionControl = (props: ConditionProps) => {
   };
 
   const deleteCondition = () => {
-    updateCondition(EMPTY_FIELD_CONDITION);
+    if (props.onChange) {
+      props.onChange(EMPTY_FIELD_CONDITION);
+    }
   };
```

Comment:
Will this lose sync between the condition useState and the parent props.onChange ? (Not that it should have both!!!)